.github/workflows/gcloud_ssh.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
Can we add a comment documenting the purpose of this script?  I think it's to run a command on the gcloud instance, whereas `gcloud compute scp` later on just transfers files

Reply:
Fixed, and added comments for all scripts/commands.